src/float/surface_transceiver/surface_transceiver.ino

```diff
@@ -21,13 +21,12 @@ bool printRFStatus = true;
 // Singleton instance of the radio driver
 RH_RF95 rf95(RFM95_CS, RFM95_INT);
 
-const uint8_t LEGAL_COMMANDS_LEN = 5;
-const String LEGAL_COMMANDS[LEGAL_COMMANDS_LEN] = {"submerge", "suck", "pump", "stop", "return"};
+const String LEGAL_COMMANDS[] = {"submerge", "suck", "pump", "stop", "return"};
```

Comment:
@benjaminwp18 Should this be a String enum maybe?

Reply:
That's something that the solution to the new float ticket would implement if necessary